### PR TITLE
[Misc clean-up] Don't attach custom event functions to window namespace

### PR DESCRIPTION
**Pull request by bbecquet**
## Description
A piece of tech clean-up I wanted to do for a long time.
We declare functions to fire/listen to custom events, but they are declared as globals, attached to the `window` object.
There is no need to do that. This pollutes the global namespace and requires special ESLint config, whereas JS modules provide the right way to import that as needed.
Also, if we need to do server side rendering with our React components, any use of `window` will break.

### CODE REVIEW

File: src/libs/customEvents.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+function fire(name, params, ...additionalParams) {
```
Comment (G-Ray) [inline src/libs/customEvents.js:2]: I think we can remove `additionalParams`, I don't see why `params` would not be enough

File: src/libs/listen.js [hunk 2]
```diff
@@ -1,26 +0,0 @@
-/**
```
Comment (G-Ray) [inline src/libs/listen.js:1]: :+1:

File: src/libs/customEvents.js (hunk 1 above)
Comment (bbecquet) [reply src/libs/customEvents.js:2]: Indeed. I just copied the previous code but this is worth simplifying. Thank you!

**Comment by amatissart** [comment]
A minor conflict needs to be resolved on this PR.

**Pull request merged**